src/main/java/net/sf/rails/algorithms/NetworkVertex.java

```diff
@@ -260,12 +260,12 @@ public boolean initRailsVertex(PublicCompany company) {
         // define locationName
         stopName = null;
         if (station.getType() == Station.Type.OFFMAPCITY) {
-            if (hex.getStopName() != null && !hex.getStopName().equals("")) {
+            if (hex.getStopName() != null && !"".equals(hex.getStopName())) {
```

Comment:
Maybe better do a size comparison instead? Or use a method from apache-commons

Reply:
Yeah, would be better to move this to using something like `StringUtils.isNotBlank()`. I didn't really spend time to review the code for a better way to implement, just migrated it